utils/classNames.js

```diff
@@ -0,0 +1,9 @@
+export default function(...args) {
+  return args.reduce((prev, index) => {
+    const [className, condition = true] = index;
```

Comment:
what does condition mean?

Reply:
so i considered the case that 
```
  const row = classNames([`row`]);
```
that means the condition (above) is `undefined`. When is `undefined` initial value is true.

I think the moment you know what this does actually makes the code cleaner. 
There are ofc many ways to do that so if you wanna go with some other way please let me know

Reply:
so, if the className doesn't exist, then add it? Seems strange?